source/NetworkUtils.cpp

```diff
@@ -270,4 +270,169 @@ void NetworkUtils::broadcastStartSession(ENetHost* serverHost) {
 	outBuffer.writeUInt16(1);	// Dummy frame num (this is where we sync and reset the client frame numbers)
 	ENetPacket* packet = enet_packet_create((void*)outBuffer.getCArray(), outBuffer.length(), ENET_PACKET_FLAG_RELIABLE);
 	enet_host_broadcast(serverHost, 0, packet);
-}
\ No newline at end of file
+}
+
+int NetworkUtils::broadcastPlayerConfigToClients(ENetHost* serverHost, const shared_ptr<PlayerEntity>& player)
+{
+	BinaryOutput outBuffer;
+	outBuffer.setEndian(G3D_BIG_ENDIAN);
+	outBuffer.writeUInt8(NetworkUtils::MessageType::SEND_PLAYER_CONFIG_TO_CLIENTS);
+	outBuffer.writeUInt16(0);	// Dummy frame num (haven't started a trial yet)
+
+	if (*player->readFromFile == false) {
```

Comment:
I don't believe this if condition is necessary, I'd choose whether to use the current `PlayerConfig` or the one from file and then just have one set of writes to `outBuffer` below instead of duplicating all the code.

Reply:
Thank you for your suggestion. I updated it.